docs/1000.yaml

```diff
@@ -3605,6 +3732,9 @@ Q10942247:
 
 Q11352023:
   title: Vitali convergence theorem
+  decl: MeasureTheory.tendstoInMeasure_iff_tendsto_Lp
+  author: Igor Khavkine
```

Comment:
I formalized Vitali convergence in https://github.com/leanprover-community/mathlib3/pull/12408 ...

Reply:
but only the finite measure version, right? Maybe we can reference both versions

Reply:
My bad, I did not realize the version using `unif_tight` is also called Vitali convergence. In that case the authorship is accurate and I'm happy to keep it as it is.